core/domain/feedback_services.py

```diff
@@ -400,7 +400,7 @@ def _get_thread_from_model(thread_model):
     Returns:
         FeedbackThread. The corresponding FeedbackThread domain object.
     """
-    if thread_model.message_count is None:
+    if not thread_model.message_count:
```

Comment:
I think it is better to do the is None check.

Reply:
But message_count is an integer. The None check won't work here

Reply:
I don't understand. So when will "not thread_model.message_count" be false?

Reply:
When message count is>0?